package.json

```diff
@@ -75,9 +75,9 @@
     "intl-messageformat": "2.2.0",
     "jquery": "=3.2.1",
     "object-assign": "4.1.1",
-    "patternfly": "3.48.1",
+    "patternfly": "3.53.1",
```

Comment:
the build is failing on node6. Looks like it's from this upgrade.

Reply:
+1.

@bond95 please extract this change into a separate PR.
cc @michalskrivanek